tools/osx/crosstool/BUILD.toolchains

```diff
@@ -7,7 +7,10 @@ exports_files(["osx_archs.bzl"])
 # Target constraints for each arch.
 # TODO(apple-rules): Rename osx constraint to macOS.
 OSX_TOOLS_CONSTRAINTS = {
-    "armeabi-v7a": ["@platforms//cpu:armv7"],
+    "armeabi-v7a": [
+        "@platforms//cpu:armv7",
+        "@platforms//os:android",
```

Comment:
This can't be right, shouldn't this be osx?

Reply:
IIUC this CPU is just a shim because the android SDK requires it to exist even if it isn't valid. With the macOS toolchain it will actually just generate a dummy toolchain that does nothing. https://github.com/bazelbuild/bazel/pull/13449

I tried to remove it entirely again recently, but it's still hit in some cases https://github.com/bazelbuild/bazel/pull/14872

Reply:
I see, thanks for explaining.